.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
```

Comment:
This is an open source repo where anyone can submit a PR so I don't think this will make sense for everyone. Maybe you could replace it with some template text like [Link to issue here if relevant]